server/index.js

```diff
@@ -4,17 +4,18 @@ const app = express();
 const path = require('path');
 const port = 3000;
 
```

Comment:
General comment: You'll need to change some code in `index.js` to dynamically render products based on the url, but it shouldnt be too bad.  There was some discussion on slack about how to do it.

Reply:
yeah I'm aware of this as well